src/main/resources/operatorhub/mta-operator/0.0.1/manifests/windup-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: windup-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+     [
+      {
+        "apiVersion": "windup.jboss.org/v1",
+        "kind": "Windup",
+        "metadata": {
+          "namespace": "mta",
+          "name": "mta-example",
+          "labels": {
+            "application": "windup"
+          }
+        }
+      }
+     ]
+    capabilities: Basic Install
+    categories: Monitoring
```

Comment:
Is it possible to change this to "Migrations"?

Reply:
yes

Reply:
yes